packages/rule-handler/src/filter.ts

```diff
@@ -66,6 +66,7 @@ export const search = async (
         headers: {
           Cookie: `auth=${auth};`,
           'Content-Type': 'application/json',
+          'X-OmnivoreClient': 'rule-handler',
```

Comment:
`client`  set to `X-OmnivoreClient` in middleware?

Reply:
Yeah, we set the `client` in the `httpContext` in middler

Reply:
```
  // set client info in the request context
  app.use(httpContext.middleware)
  app.use('/api/', (req, res, next) => {
    const client = req.header('X-OmnivoreClient')
    if (client) {
      httpContext.set('client', client)
    }
    next()
  })
```